app/panel/components/Header.jsx

```diff
@@ -137,7 +150,9 @@ class Header extends React.Component {
 			active: this.props.is_expert,
 		});
 		const { loggedIn, user } = this.props;
+		const subscriber = isSubscriber(user);
 		const rightLink = this.generateLink();
+		const badgeClasses = `columns shrink ${(!loggedIn || !subscriber) ? 'non-subscriber-badge' : 'subscriber-badge'}`;
```

Comment:
@zarembsky let's use the `classnames` library for these dynamic classes

Reply:
OK